ui.apps/src/main/content/jcr_root/apps/core/cif/components/content/button/v1/button/.content.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:Component"
+    jcr:title="Button"
+    componentGroup="CIF Core Components"/>
```

Comment:
We should in use WCM core button component as super type.

Reply:
Component is extended using WCM core component and resourceSuperType is used